src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -140,6 +169,21 @@ public static List<HearingLocation> getHearingLocations(SscsCaseData caseData,
             .collect(Collectors.toCollection(ArrayList::new));
     }
 
+    private static String getVenueID(SscsCaseData caseData, String nextHearingVenue) throws InvalidMappingException {
+        if (SOMEWHERE_ELSE.getValue().equals(nextHearingVenue)) {
+            return caseData.getAdjournCaseNextHearingVenueSelected().getValue().getCode();
+        }
+
+        Hearing latestHearing = caseData.getLatestHearing();
+
+        if (nonNull(latestHearing)) {
+            return latestHearing.getValue().getVenueId();
```

Comment:
Tmportant: this should be:
```suggestion
            return latestHearing.getValue().getEpimsId();
```

Reply:
Can you also change the references to VenueID to epimsId in the method name, variables and logs, as venueId is a gaps value and will get confusing

Reply:
Done

Reply:
Done